stock_outgoing_shipment_report/views/stock_outgoing_shipment_report_views.xml

```diff
@@ -6,9 +6,14 @@
         <field name="arch" type="xml">
             <search string="Outgoing Moves">
                 <field name="carrier_id" />
+                <field name="create_date" />
                 <group expand="0" string="Group By">
                     <filter name="group_by_carrier" string="Carrier" context="{'group_by':'carrier_id'}"/>
                 </group>
+                <group expand="0" string="Group By">
+                    <filter name="group_by_create_date" string="Create date" context="{'group_by':'create_date'}"/>
+                    <filter name="search_by_create_date" string="Create date" context="{'search':'create_date'}"/>
```

Comment:
This filter doesn't make sense to me.  You need to update this record https://github.com/qrtl/hls-custom/blob/0d6008b201a8aa204e05c5e7dea74d90c4475050/stock_outgoing_shipment_report/views/stock_outgoing_shipment_report_views.xml#L31-L37 to add, for example, `<field name="context">{'search_default_created_today': 1}</field>` upon creating the `created_today` filter, in case you are lost what to do.